packages/patterns/src/hasGradient.ts

```diff
@@ -15,12 +15,12 @@
  */
 
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { Pattern, PatternType } from './types';
+import { Pattern, PatternType } from '@googleforcreators/types';
 
 function hasGradient(pattern: Pattern) {
-  return Boolean(pattern?.type) && PatternType.SOLID !== pattern.type;
+  return Boolean(pattern?.type) && PatternType.Solid !== pattern.type;
```

Comment:
Curious why change from uppercase to camelcase? I can see it's still uppercase for `ResourceType` for example, should we do just one or the other for consistency?

Reply:
[TypeScript enums are generally written in `PascalCase`](https://basarat.gitbook.io/typescript/styleguide#enum), so let's try to stick to that. I'll update `PatternType` as well.

Reply:
Renamed the ResourceType members too.